.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
lets also add a check for the ruby dependencies too

Reply:
Can you help me with this?

Reply:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem list all supported package-ecosystem add another section for the ruby one